src/helpers/interactionHandler.ts

```diff
@@ -0,0 +1,43 @@
+import { AutocompleteInteraction, CommandInteraction, SelectMenuInteraction } from "discord.js"
+import { logger } from "../logger"
+import { ClientType } from "../types"
+import { DeleteLinkById } from "./linkQueries"
+
+const HandleSelectMenu = async (interaction: SelectMenuInteraction) => {
+    if (interaction.customId === 'delete-confirmation') {
+        let answer = interaction.values[0]
+        if (answer == "No") {
+            await interaction.update({ content: "Good call, boss! Your link is safe.", components: [] });
+        } else {
+            await DeleteLinkById(answer)
+            await interaction.update({ content: "Deleted! You're welcome :)", components: [] });
+        }
+    }
+}
+const HandleAutoComplete = async (client: ClientType, interaction: AutocompleteInteraction) => {
+    const command = client.commands.get(interaction.commandName);
+    if (!command?.autoComplete) return;
+
+    try {
+        await command.autoComplete(interaction);
+    } catch (error) {
+        logger.debug("Autocomplete Error:", error);
```

Comment:
This should be an error flag instead of debug

Reply:
Its important to remember that debug flags are turned off during production.

Reply:
ok got it. Didn't know this before 😅